radicle-cli/src/commands/patch/common.rs

```diff
@@ -77,6 +77,7 @@ pub fn get_merge_target(
     project: &Project,
     storage: &Repository,
     head_branch: &git::raw::Branch,
+    options: &Options,
```

Comment:
Since we're only using the `target` in `Options`, I'd suggest passing that in:
```suggestion
    target: Option<&NodeId>,
```